release_notes.md

```diff
@@ -18,6 +18,14 @@ This file tracks release notes for the loop client.
 
 #### Breaking Changes
 
+In loopd.conf file `maxlsatcost` and `maxlsatfee` were renamed to `maxl402cost`
+and `maxl402fee` accordingly. If they have been changed locally, the file has
+to be updated for loopd to recognize the options.
+
+The path in looprpc "/v1/lsat/tokens" was renamed to "/v1/l402/tokens" and
```

Comment:
Hmm, okay, so we warn about breaking changes here. So I guess it's up to the team to decide whether that's enough or we want to keep backward compatibility as suggested in the comments above.

Reply:
I updated the release notes. Now the only broken scenario that I found is `loop listauth` if the `loop` binary is from a previous release, see above.